opencti-platform/opencti-graphql/tests/02-integration/02-resolvers/indicator-test.ts

```diff
@@ -1,6 +1,6 @@
 import { expect, it, describe } from 'vitest';
 import gql from 'graphql-tag';
-import { queryAsAdmin } from '../../utils/testQuery';
+import { adminQuery, queryAsAdmin } from '../../utils/testQuery';
```

Comment:
wondering: is it because queryAsAdmin was not working here ?

Reply:
I don't know... I had so many problems with `queryAsAdmin` which is just a test method which does not take the context into account that I prefer use `adminQuery`